web/src/main/java/com/epam/rft/atsy/web/controllers/rest/CandidateController.java

```diff
@@ -28,65 +27,56 @@
 @RequestMapping(value = "/secure/candidates")
 public class CandidateController {
 
-  public static final String NAME = "name";
-  public static final String EMAIL = "email";
-  public static final String PHONE = "phone";
-
-  private static final String EMPTY_JSON = "{}";
   private static final Logger LOGGER = LoggerFactory.getLogger(CandidateController.class);
+
+  private static final SearchOptions
+      EMPTY_SERCH_OPTIONS =
+      SearchOptions.builder().name(StringUtils.EMPTY).email(StringUtils.EMPTY)
+          .phone(StringUtils.EMPTY).build();
+
   @Resource
   private CandidateService candidateService;
 
-  /**
-   * Loads the page according to the given parameters.
-   * @param filter Non-required parameter which represents the requested filtering
-   * @param order required parameter which represents the requested ordering (ascending/descending)
-   * @param sortField required parameter which represents which field is used in sorting
-   * @return A collection of CandidateDTO objects which is sorted and filtered according to the
-   * given parameters
-   */
+  //TODO: Add Java Doc.
   @RequestMapping(method = RequestMethod.GET)
-  public Collection<CandidateDTO> loadPage(
+  public PagingResponse<CandidateDTO> loadPage(
       @RequestParam(value = "filter", required = false) String filter,
-      @RequestParam("order") String order,
-      @RequestParam("sort") String sortField) {
+      @RequestParam(value = "sortName", required = true) String sortName,
+      @RequestParam(value = "sortOrder", required = true) String sortOrder,
+      @RequestParam(value = "pageSize", required = true) int pageSize,
+      @RequestParam(value = "pageNumber", required = true) int pageNumber) {
 
-    SortingRequest.Order
-        orderDirection =
-        SortingRequest.Order.valueOf(StringUtils.upperCase(order));
-    SortingRequest.Field fieldName = SortingRequest.Field.valueOf(StringUtils.upperCase(sortField));
+    //It is required because bootstraps first page is 1
+    pageNumber--;
 
-    FilterRequest filterRequest = FilterRequest.builder()
-        .order(orderDirection)
-        .fieldName(fieldName)
-        .searchOptions(parseFilters(filter))
-        .build();
+    SearchOptions searchOptions = parseFilters(filter);
+    String name = MoreObjects.firstNonNull(searchOptions.getName(), StringUtils.EMPTY);
+    String email = MoreObjects.firstNonNull(searchOptions.getEmail(), StringUtils.EMPTY);
+    String phone = MoreObjects.firstNonNull(searchOptions.getPhone(), StringUtils.EMPTY);
+    String position = MoreObjects.firstNonNull(searchOptions.getPosition(), StringUtils.EMPTY);
```

Comment:
Minor improvement suggestion: local variables (name, email, phone, position, candidateFilterRequest) can be final.